frontend/src/components/AdminControls/Recipes.js

```diff
@@ -33,6 +43,7 @@ const Recipes = ({ recipes }) => {
   );
 
   const recipeRow = ({
+    id,
```

Comment:
I think you need to pass this into `adminButtons` below on line 57. That's what CircleCI/the linter is choking on.